src/clients/java/src/test/java/com/tigerbeetle/IntegrationTest.java

```diff
@@ -699,6 +699,103 @@ public void testCreatePendingTransfersAndVoid() throws Throwable {
         }
     }
 
+    @Test
+    public void testCreatePendingTransfersAndVoidExpired() throws Throwable {
+
+        try (var server = new Server()) {
```

Comment:
Drive-by, but can’t we take advantage of using declaration here and save an indent?

Reply:
🤦  this is Java, not C#

Reply:
Though, we should be able to put both the server and the client into a single try?

```
try (FileReader fr = new FileReader(path);
	         BufferedReader br = new BufferedReader(fr)) {
	        return br.readLine();
	    }
```

Reply:
And I think we don’t need catch here?

Reply:
TIL it's possible to have more than one object inside a Java try-resource block!!

> And I think we don’t need catch here?

Yes, since we don't throw on `close` anymore, we can remove this catch block!